samples/ControlCatalog/Pages/PointerCanvas.cs

```diff
@@ -132,20 +132,20 @@ protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
             if (_stopwatch.Elapsed.TotalMilliseconds > 250)
             {
                 Status = $@"Events per second: {(_events / _stopwatch.Elapsed.TotalSeconds)}
-PointerUpdateKind: {_lastProperties?.PointerUpdateKind}
-Last PointerUpdateKind != Other: {_lastNonOtherUpdateKind}
-IsLeftButtonPressed: {_lastProperties?.IsLeftButtonPressed}
-IsRightButtonPressed: {_lastProperties?.IsRightButtonPressed}
-IsMiddleButtonPressed: {_lastProperties?.IsMiddleButtonPressed}
-IsXButton1Pressed: {_lastProperties?.IsXButton1Pressed}
-IsXButton2Pressed: {_lastProperties?.IsXButton2Pressed}
-IsBarrelButtonPressed: {_lastProperties?.IsBarrelButtonPressed}
-IsEraser: {_lastProperties?.IsEraser}
-IsInverted: {_lastProperties?.IsInverted}
-Pressure: {_lastProperties?.Pressure}
-XTilt: {_lastProperties?.XTilt}
-YTilt: {_lastProperties?.YTilt}
-Twist: {_lastProperties?.Twist}";
+                            PointerUpdateKind: {_lastProperties?.PointerUpdateKind}
+                            Last PointerUpdateKind != Other: {_lastNonOtherUpdateKind}
+                            IsLeftButtonPressed: {_lastProperties?.IsLeftButtonPressed}
+                            IsRightButtonPressed: {_lastProperties?.IsRightButtonPressed}
+                            IsMiddleButtonPressed: {_lastProperties?.IsMiddleButtonPressed}
+                            IsXButton1Pressed: {_lastProperties?.IsXButton1Pressed}
+                            IsXButton2Pressed: {_lastProperties?.IsXButton2Pressed}
+                            IsBarrelButtonPressed: {_lastProperties?.IsBarrelButtonPressed}
+                            IsEraser: {_lastProperties?.IsEraser}
+                            IsInverted: {_lastProperties?.IsInverted}
+                            Pressure: {_lastProperties?.Pressure}
+                            XTilt: {_lastProperties?.XTilt}
+                            YTilt: {_lastProperties?.YTilt}
+                            Twist: {_lastProperties?.Twist}";
```

Comment:
I think this formatting is incorrect. It will actually insert blankspaces starting with line 2.